src/components/GitPanel.tsx

```diff
@@ -842,7 +842,7 @@ export class GitPanel extends React.Component<IGitPanelProps, IGitPanelState> {
    * @param path - repository path
    */
   private async _hasIdentity(path: string | null): Promise<string | null> {
-    if (!path) {
+    if (path == null) {
```

Comment:
The liner suggests to use triple equals here. The argument annotation says it cannot be undefined so it should be equivalent:


```suggestion
    if (path === null) {
```

Unless this fix depends on it, then we need to change linter eqeqeq rule to smart and add undefined in the argument.

Also, can you add a test?